google/cloud/storage/blob.py

```diff
@@ -171,6 +171,10 @@ def __init__(
         kms_key_name=None,
         generation=None,
     ):
+        """
+        property :attr:`name`
+            Get the blob's name.
+        """
```

Comment:
I never seen such a way of documenting properties. Does googlers use something like that?

Reply:
Yes, but for that need to create a getter setter method like use `@property` decorator which breaks the current functionality, could you suggest anything here?

Reply:
I agree with @IlyaFaer , this is quite an unusual look for Google style. Would it not be a better way to use line comments instead?

Reply:
@IlyaFaer @mf2199  Yes i also agree, but not able to find other way. Could you guys suggest anything or should i ask Frank about it?

Reply:
First of all, here `name` is simply a local variable, whereas Frank explicitly calls it a property. Thus it can be converted to such, e.g.:
```
def __init__(self, name, ...)
    ...
    self._name = name
    ...

@property
def name(self):
   """ Get the blob's name"""
   return self._name
```

Reply:
@mf2199 it throws `AttributeError: can't set attribute` , even can't expose setter method.